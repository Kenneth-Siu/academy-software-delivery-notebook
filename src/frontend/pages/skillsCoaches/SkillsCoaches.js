import React from "react";
import "./SkillsCoaches.scss";

export default function SkillsCoaches() {
    return (
        <main className="skills-coaches-page">
            <h1>Skills Coaches</h1>
        </main>
    );
}
