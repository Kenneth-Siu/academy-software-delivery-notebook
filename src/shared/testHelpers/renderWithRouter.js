import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

export default function renderWithRouter(component) {
    return render(<MemoryRouter>{component}</MemoryRouter>);
}
