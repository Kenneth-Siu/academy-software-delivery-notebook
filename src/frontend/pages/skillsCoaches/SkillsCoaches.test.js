import React from "react";
import renderWithRouter from "../../../shared/testHelpers/renderWithRouter";
import SkillsCoaches from "./SkillsCoaches";

describe("SkillsCoaches", () => {
    test("renders", () => {
        const tree = renderWithRouter(<SkillsCoaches />);
        expect(tree.container).toMatchSnapshot();
    });
});
