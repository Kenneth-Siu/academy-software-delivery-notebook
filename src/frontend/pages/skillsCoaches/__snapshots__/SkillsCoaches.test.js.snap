// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkillsCoaches renders 1`] = `
<div>
  <main
    class="skills-coaches-page"
  >
    <h1>
      Skills Coaches
    </h1>
  </main>
</div>
`;
